src/Umbraco.Web/Mvc/UmbracoViewPageOfTModel.cs

```diff
@@ -235,6 +236,18 @@ public override void WriteLiteral(object value)
             base.WriteLiteral(value);
         }
 
+        public override void Write(object value)
+        {
+            if (value is IHtmlEncodedString)
+            {
+                base.WriteLiteral(value);
```

Comment:
Should this be `value.ToHtmlString()` ? Since that is what the `System.Web.HttpUtility.HtmlEncode` is doing when writing out these values

Reply:
I do that because this is safer. The only reason it worked now is that `ToString` and `ToHtmlString` is doing the same.